app/controllers/templates_controller.rb

```diff
@@ -95,6 +95,10 @@ def preview
     safe_render(@template)
   end
 
+  def export
+    send_data @template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @template.filename
```

Comment:
I'd suggest putting this in a concern to prevent duplication across all Template-subclass controllers.